bigtable/google/cloud/bigtable/table.py

```diff
@@ -29,6 +29,18 @@
 from google.cloud.bigtable.row_data import PartialRowsData
 
 
+# Maximum number of mutations in bulk
+_MAX_BULK_MUTATIONS = 100000
```

Comment:
Then the comment should reference that URL (e.g., for verification, or if the back-end should change the limit in the future).